apps/builtin/README.md

```diff
@@ -0,0 +1,43 @@
+# builtin
+
+The **builtin** supports registering a TASH command through Makefile without codes change.  
```

Comment:
The **builtin** supports registering commands in TASH through Makefile without calling any TASH public api functions. **builtin** helps in porting the application directly to tinyara without changing application's code .